test/addons/rook-pool/storage-class.yaml

```diff
@@ -7,7 +7,7 @@ kind: StorageClass
 metadata:
     name: rook-ceph-block
     labels:
-        ramendr.openshift.io/storageid: rook-ceph-storage-id
+        ramendr.openshift.io/storageid: rook-ceph-storage-id-on-$cluster
```

Comment:
`rook-ceph-storage-id` is a good name for placeholder id, something to use when we don't have a real storage id. Now that we have a storage id, I think that something like `rook-ceph-{cluster}-1` is a better name, so when we add a second storage on the same cluster we can name it `rook-ceph-{cluster}-2`.